### DISCUSSION TITLE
cod zombies Easter Egg songs pack (PC, 3ds/psp support)

### DISCUSSION THREAD

**Discussion by ToonLink0001** [nzp-team/nzportable#716] [Other Content Releases]
This mod replaces the Easter eggs with the official ones for Call of Duty Zombies. This mod pack also contains a replacement Easter egg song for the new Kino map release.

I DO NOT OWN THESE SONGS ALL RIGHTS GO TO THE ORIGINAL CREATORS OF THESE SONGS

to add them, go to the tracks folder and replace all of the files in there with the ones from this pack and your all set!

Songs:
Beyond me -> beauty of annihilation by Kevin Sherwood and Elena Sigmand
Hanger -> Shepherd of fire By Avenged sevenfold
Surface tension -> Where are we going? (Bo4 remix) by Kevin Sherwood
Once more -> Won't Back Down by Eminem
chipped 115 -> 115 by Kevin Sherwood and Elena Sigmand

3DS and PSP SUPPORT UPDATE, if you just want the songs for the 3DS/PSP, you can delete the files that have. OGG or. WAV,

[>**PC DOWNLOAD**<](https://drive.google.com/file/d/18ac6AhWhp4qZ1n-kb6Qi7PPkU74gfFTG/view)
[**3DS/PSP**](https://drive.google.com/file/d/1KqMqOtmZ7rTLfOq9MTesZpi39YXmbVD0/view)

**Comment by ToonLink0001** [comment]
planning to replace wont back down with beauty of annihilation

**Reply by kmorris1077** [reply]
The google drive link 404s

**Reply by ToonLink0001** [reply]
ok thanks ill fix it

**Reply by ToonLink0001** [reply]
link fixed

**Comment by ToonLink0001** [comment]
NOTE: GOOGLE DRIVE LINK WORKS AGAIN

**Reply by curtmanps4-gif** [reply]
Now it doesn't work again
